Parser/Client/Browser.php

```diff
@@ -92,9 +93,11 @@ class Browser extends AbstractClientParser
         'A5' => 'Amerigo',
         'AO' => 'Amigo',
         'AN' => 'Android Browser',
+        '3A' => 'AOL',
```

Comment:
According to https://browserwars.fandom.com/wiki/AOL_Explorer and https://en.wikipedia.org/wiki/AOL_Explorer (which clearly indicates Trident engine) and some screenshots I was able to find, it looks like this should be `AOL Explorer` or, at least `AOL Browser`